packages/application-shell/src/components/app-bar/app-bar.js

```diff
@@ -1,43 +1,93 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { Link } from 'react-router-dom';
-import { Spacings } from '@commercetools-frontend/ui-kit';
+import { css } from '@emotion/core';
+import { Spacings, customProperties } from '@commercetools-frontend/ui-kit';
 import LogoSVG from '@commercetools-frontend/assets/images/logo.svg';
 import UserSettingsMenu from '../user-settings-menu';
 import ProjectSwitcher from '../project-switcher';
 import BackToProject from '../back-to-project';
 import { getPreviousProjectKey } from '../../utils';
 import LoadingPlaceholder from '../loading-placeholder';
 import { REQUESTS_IN_FLIGHT_LOADER_DOM_ID } from '../requests-in-flight-loader/constants';
-import styles from './app-bar.mod.css';
 
 const AppBar = props => {
   const previousProjectKey = getPreviousProjectKey(
     props.user && props.user.defaultProjectKey
   );
 
   return (
-    <div className={styles['app-bar']} data-test="top-navigation">
+    <div
+      css={css`
+        box-shadow: ${customProperties.shadow1};
+        min-height: 43px;
+        position: relative;
+        width: 100%;
+        z-index: 20000;
+        display: flex;
+        align-items: center;
+        justify-content: space-between;
+      `}
+      data-test="top-navigation"
+    >
       <Spacings.Inline>
-        <div className={styles.logo}>
+        <div
+          css={css`
+            display: block;
+            float: left;
+            width: 28px;
+            height: 32px;
+            margin-left: 20px;
+          `}
+        >
           {!props.user ? (
-            <img src={LogoSVG} className={styles['logo-img']} alt="Logo" />
+            <img src={LogoSVG} width="100%" alt="Logo" />
           ) : (
             <Link to={`/${previousProjectKey || ''}`}>
-              <img src={LogoSVG} className={styles['logo-img']} alt="Logo" />
+              <img src={LogoSVG} width="100%" alt="Logo" />
             </Link>
           )}
         </div>
 
         <div
           id={REQUESTS_IN_FLIGHT_LOADER_DOM_ID}
-          className={styles['loader-container']}
+          css={css`
+            display: flex;
+            flex: 1;
+          `}
         />
       </Spacings.Inline>
 
-      <div className={styles.navigation}>
+      <div
+        css={css`
+          float: right;
+          font-weight: normal;
+          font-size: 1rem;
+          margin-right: ${customProperties.spacingM};
+          display: flex;
+          align-items: center;
+        `}
+      >
         <Spacings.Inline scale="m" alignItems="center">
-          <div className={styles.switchers}>
+          <div
+            css={css`
+              /* Special sizes for dropdowns */
+              .Select-input,
```

Comment:
why do we need these react-select v1 custom styles?

Reply:
I just used what was already there. Do you think we don't need them anymore?

Reply:
Yeah. That's all react-select v1 css app-kit doesn't use react-select anymore (it just uses the ui-kit wrappers for react select v3)

Reply:
Cool, thanks for the reminder!